Validator/Constraints/PhoneNumberValidator.php

```diff
@@ -30,6 +31,10 @@ class PhoneNumberValidator extends ConstraintValidator
      */
     public function validate($value, Constraint $constraint)
     {
+        if (!$constraint instanceof PhoneNumber) {
+            throw new UnexpectedTypeException($constraint, __NAMESPACE__ . '\PhoneNumberValidator');
```

Comment:
```php
throw new UnexpectedTypeException($constraint, __NAMESPACE__ . '\PhoneNumber');
```

Reply:
or just pass `PhoneNumber::class` as the second argument

Reply:
I did this to follow the convention of the built-in Symfony constraint validators, would you like me to change it anyway?

Reply:
I have no preference regarding an hardcoded string or using the class constant (in Symfony, this is for historical reasons because of the PHP 5.3 support). However, you will of course have to drop the `Validator` suffix.

Reply:
We have to supporto PHP 5.3 here too, unfortunately according to the `composer.json` 😞

Reply:
That’s right - unfortunately Symfony supports 5.3 even in 2.8, so we should try to maintain 5.3 support until there’s a strong reason not to.

Reply:
>we should try to maintain 5.3 support until there’s a strong reason not to.

A strong reason is that PHP 5.3 is not supported anymore since 2014, but I understand this topic is out of scope for this PR and should not be addressed here

Reply:
Well, yep - that'd be a strong reason for Sensio to drop 5.3 from Symfony's LTS commitment. Unluckily for us they decided to keep it alive in the Symfony ecosystem - as a library there's not much we can do about that.